nx-X11/programs/Xserver/os/Imakefile

```diff
@@ -121,16 +124,51 @@ TMEMCMP_OBJS = timingsafe_memcmp.o
 #endif
 
 BOOTSTRAPCFLAGS = 
-           SRCS = WaitFor.c access.c connection.c io.c $(COLOR_SRCS) \
-                  osinit.c utils.c log.c auth.c mitauth.c secauth.c \
-                  $(XDMAUTHSRCS) $(RPCSRCS) xdmcp.c OtherSources \
-                  xstrans.c $(SNPRINTF_SRCS) $(STRLCAT_SRCS) \
-		  $(REALLOCARRAY_SRCS) xprintf.c $(TMEMCMP_SRCS)
-           OBJS = WaitFor.o access.o connection.o io.o $(COLOR_OBJS) \
-                  osinit.o utils.o log.o auth.o mitauth.o secauth.o \
-                  $(XDMAUTHOBJS) $(RPCOBJS) xdmcp.o OtherObjects \
-                  xstrans.o $(SNPRINTF_OBJS) $(STRLCAT_OBJS) \
-		  $(REALLOCARRAY_OBJS) xprintf.o $(TMEMCMP_OBJS)
+           SRCS = WaitFor.c		\
+                  access.c		\
+                  connection.c		\
+                  io.c			\
+                  $(COLOR_SRCS)		\
+                  osinit.c		\
+                  utils.c		\
+                  log.c			\
+                  auth.c		\
+                  mitauth.c		\
+                  secauth.c		\
+                  $(XDMAUTHSRCS)	\
+                  $(RPCSRCS)		\
+                  xdmcp.c		\
+                  OtherSources		\
+                  xstrans.c		\
+                  $(SNPRINTF_SRCS)	\
+                  $(STRLCAT_SRCS)	\
+                  xprintf.c		\
```

Comment:
> Try to keep the order. In the original string, xprintf.c comes after $(REALLOCARRAY_SRCS). This isn't a problem per se, but in the OBJS definition, the ordering is different again: first $(REALLOCARRAY_OBJS), then $(TMEMCMP_OBJS), then xprintf.o. Please try to keep both definitions in synch, since humans are pretty good at pattern matching and would expect entries to be at the same relative location if all the others are. That is, I don't care about the actual order, but I'd like to have a consistent one.
>
> In both cases, too, $(REALLOCARRAY_SRCS) is followed by spaces, whereas you converted all the other lines to use tabs.

Nice catch! Fixed.